test/suites/crud_server.lua

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env tarantool
+
+local os = require('os')
```

Comment:
`require('os')` is excessive since `os` is exposed to global in Tarantool

```
$ tarantool
Tarantool 2.10.0-beta2-283-gb56e012bc
type 'help' for interactive help
tarantool> os
---
- execute: 'function: builtin#116'
  rename: 'function: builtin#118'
  environ: 'function: 0x40be3f60'
  setenv: 'function: 0x402c8458'
  setlocale: 'function: builtin#126'
  getenv: 'function: builtin#120'
  difftime: 'function: builtin#125'
  remove: 'function: builtin#117'
  date: 'function: builtin#123'
  exit: 'function: 0x40bdd868'
  time: 'function: builtin#124'
  clock: 'function: builtin#122'
  tmpname: 'function: builtin#119'
...
```

Reply:
Done, deleted